frontend/testing/playwright/tests/process-editor/process-editor.spec.ts

```diff
@@ -147,7 +147,6 @@ test('That it is possible to create a custom receipt', async ({ page, testAppNam
   await processEditorPage.customReceiptConfig.writeLayoutSetId(newLayoutSetId);
   await processEditorPage.dataModelConfig.clickOnAddDataModelCombobox();
   await processEditorPage.dataModelConfig.chooseOption(newDataModel);
-  await processEditorPage.pressEscapeOnKeyboard();
```

Comment:
Why is this removed? 😊

Reply:
I think it was originally added due to an issue with the combobox in the receipt, where it wouldn't close after selecting a datamodel. When the issue was fixed in this PR, this escape function wasn't necessary anymore and at the same time it now made the Playwright test fail because pressing Escape while (or after) the combobox was closing removed the selected datamodel.